public/locally-available-studies.json

```diff
@@ -3,29 +3,29 @@
     {
       "name": "Ion Base Study",
       "icons": {
-        "32": "https://addons.cdn.mozilla.net/user-media/addon_icons/2644/2644632-32.png?modified=4a64e2bc",
-        "64": "https://addons.cdn.mozilla.net/user-media/addon_icons/2644/2644632-64.png?modified=4a64e2bc",
-        "128": "https://addons.cdn.mozilla.net/user-media/addon_icons/2644/2644632-128.png?modified=4a64e2bc"
+        "32": "https://addons.cdn.mozilla.net/static/img/addon-icons/default-32.png",
```

Comment:
We are currently using the default icon, this will need to change once we update that.